### PR TITLE
added configurations to debug and imported relevant env vars on init

### PR DESCRIPTION
**Pull request by jcllobet**
Added VSCode debugging config for `zoning.data_processing.eval` & initialized OpenAI key loading from `.env` in `zoning` package.

### CODE REVIEW

File: launch.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (maxdumas) [inline launch.json:1]: This `launch.json` is unnecessary, isn't it?

File: zoning/__init__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import os
+import openai
+from dotenv import load_dotenv
+
+# Get the absolute path to the .env file
+dotenv_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), ".env")
+
+# Load environment variables from .env
+load_dotenv(dotenv_path)
```
Comment (maxdumas) [inline zoning/__init__.py:15]: Minor note that you don't need to do this. `load_dotenv` will [look up the directory tree from the CWD until it finds a `.env` file](https://github.com/theskumar/python-dotenv/blob/main/src/dotenv/main.py#L337-L338) automatically and by default if you don't provide a path yourself. That would probably be more robust.

Comment (jcllobet) [reply zoning/__init__.py:15]: Getpath with utils.py

def get_project_root() -> Path:
    repo = Repo(".", search_parent_directories=True)
    path = repo.working_tree_dir

    assert path is not None, "You are not currently in a Git repo!"

    return Path(path)

**Comment by jcllobet** [comment]
addressed @maxdumas comments, should work now

**Pull request merged**